_hackbars/Mahendrasingh.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: Make Earth a better place
+author: Sai Ashish
+---
+
+“Trees are poems that the earth writes upon the sky.”
+
+---
+
+## We are losing the Earth ⏱
```

Comment:
Codacy found an issue: [[heading-style] Headings should use setext](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6289829)